lib/path.js

```diff
@@ -935,15 +935,14 @@ const win32 = {
     if (path.length >= 2) {
       const code = path.charCodeAt(0);
```

Comment:
This is unnecessary. This could all be just one `if` statement. `path.length >= 2 && path.charCodeAt(1) === CHAR_COLON && isWindowsDeviceRoot(path.charCodeAt(0))`